### PR TITLE
pass params in initRegistration

### PR DESCRIPTION
**Pull request by francisco-sanchez-molina**
Pass params to the request

### CODE REVIEW

**Comment by francisco-sanchez-molina** [comment]
Hi @scotthovestadt ! can you review it? Thanks!

**Comment by scotthovestadt** [comment]
Sorry for delay. LGTM, was obvious a mistake originally.

File: lib/accounts.ts [hunk 1]
```diff
@@ -187,8 +187,8 @@ export class Accounts {
      *
      * @see http://developers.gigya.com/display/GD/accounts.initRegistration+REST
      */
-    public initRegistration(params?: BaseParams) {
-        return this.gigya.request<AccountsInitRegistrationResponse>('accounts.initRegistration', {});
+    public initRegistration(params: BaseParams) {
```
Comment (scotthovestadt) [inline lib/accounts.ts]: make this `params: BaseParams = {}`

Comment (francisco-sanchez-molina) [reply lib/accounts.ts]: you are right! Done ;)

**Pull request merged**